lookout/style/format/debug/train.py

```diff
@@ -0,0 +1,45 @@
+"""Facilities to train a FormatModel without a lookout server."""
```

Comment:
Please put this file into `research` for now. I need to find some time to refactor it and make nice - but not this week.